specs/PermissionManagement.md

```diff
@@ -385,6 +386,11 @@ typedef enum COREWEBVIEW2_PERMISSION_KIND {
   /// when developers use the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API)
   /// to request access to system exclusive MIDI messages.
   COREWEBVIEW2_PERMISSION_KIND_MIDI_SYSTEM_EXCLUSIVE_MESSAGES,
+
+  /// Indicates permission to open and place windows on the screen. Permission is
+  /// requested when developers use the [Multi-Screen Window Placement API](https://www.w3.org/TR/window-placement/)
+  /// to get screen details.
+  COREWEBVIEW2_PERMISSION_KIND_WINDOW_MANAGEMENT,
```

Comment:
Any reason to call this 'management' rather than 'placement'? The underlying feature is placement, and management implies more than that.

Reply:
The permission in the DOM APIs is called [`window-management`](https://www.w3.org/TR/window-placement/#api-permission-policy-integration) . They renamed it from `window-placement` to `window-management`  to cover more than just placement for the API ([ref](https://github.com/w3c/window-placement/issues/114)).